packages/trace-viewer/src/ui/attachmentsTab.tsx

```diff
@@ -20,9 +20,33 @@ import { ImageDiffView } from '@web/shared/imageDiffView';
 import type { MultiTraceModel } from './modelUtil';
 import { PlaceholderPanel } from './placeholderPanel';
 import type { AfterActionTraceEventAttachment } from '@trace/trace';
+import { isTextualMimeType } from 'playwright-core/lib/utils';
```

Comment:
@yury-s I can't for the life of me figure out why this import doesn't work, the build fails.
I see the function is exported as part of utils index.ts, and VSCode correctly finds the path. However an error is thrown during the build:
```
"isTextualMimeType" is not exported by "../playwright-core/lib/utils/index.js", imported by "src/ui/attachmentsTab.tsx".
```
I'm not too familiar with the build system used in this repository so I assume I'll need to modify some additional config (tsconfig / vite related), but so far nothing I tried worked

Reply:
To fix this problem:
- move `packages/playwright-core/src/utils/mimeType.ts` to `packages/playwright-core/src/utils/isomorphic/mimeType.ts`
- update existing imports
- import it here as `import { isTextualMimeType } from '@isomorphic/mimeType'`

This is because trace viewer runs in the browser, so it cannot use Node.js-dependent code, but allowed to use isomorphic code.

Reply:
thank you @dgozman, I managed to fix it following your instructions